text/0083-multi-level-meta-data-space.md

```diff
@@ -0,0 +1,52 @@
+# Multi-level Meta Data Space
+
+## Summary
+
+PD can handle multiple users' meta data, each user only see its own meta data.
+
+## Motivation
+
+PD is the mata data center of TiKV Cluster, the meta data contains TiKV instances' addr, user data range(region)
+information, the number of replicas of data, etc.
+
+But PD currently only can handle the key space [min-key, max-key], it means that PD only can handle one user's 
+meta data, it is not possible when multiple users want share the same PD cluster as their meta system. There are
+two different scenarios that require PD can handle multiple user' meta data:
+
+1. Multiple TiKV Cluster share the same PD cluster. Because the minimal demplyment of a TiKV Cluster is 3 TiKV 3 PD,
+but it is not cost-effect if every small cluster has 3 dedicated meta data node.
+2. There are Multiple tenant in the same TiKV Cluster, each tenant has it own meta data, each tenant's key range can
```

Comment:
Is the [keyspace](https://github.com/tikv/rfcs/blob/7311fc02b21d3e5dc10165192f60cdc4610e98a3/text/0069-api-v2.md#keyspace-management) in API v2 match this?

Reply:
No, v2 API can not satisfy multiple TiDB tenants.

Reply:
When there are multiple TiDB tenants, each TiDB should has its own `ddl-owner`, `gc-safepoint` and other meta data, these meta data should be stored in PD separately.